third_party/accessibility/ax/platform/ax_platform_node.h

```diff
@@ -5,6 +5,7 @@
 #ifndef UI_ACCESSIBILITY_PLATFORM_AX_PLATFORM_NODE_H_
 #define UI_ACCESSIBILITY_PLATFORM_AX_PLATFORM_NODE_H_
 
+#include <functional>
```

Comment:
same here

Reply:
the header uses `std::function` which is declared in `<functional>`. Not sure why other platforms don't seem to need it.